scene/gui/text_edit.cpp

```diff
@@ -729,15 +730,14 @@ void TextEdit::_notification(int p_what) {
 					VisualServer::get_singleton()->canvas_item_add_rect(ci, Rect2(xmargin_beg, ofs_y, xmargin_end - xmargin_beg, get_row_height()), cache.mark_color);
 				}
 
-				if (text.is_breakpoint(line)) {
-
-					VisualServer::get_singleton()->canvas_item_add_rect(ci, Rect2(xmargin_beg, ofs_y, xmargin_end - xmargin_beg, get_row_height()), cache.breakpoint_color);
-				}
-
 				if (line == cursor.line) {
 					VisualServer::get_singleton()->canvas_item_add_rect(ci, Rect2(0, ofs_y, xmargin_end, get_row_height()), cache.current_line_color);
 				}
 
+				if (text.is_breakpoint(line) && !draw_breakpoint_gutter) {
+					VisualServer::get_singleton()->canvas_item_add_rect(ci, Rect2(xmargin_beg, ofs_y + get_row_height() - EDSCALE, xmargin_end - xmargin_beg, EDSCALE), cache.breakpoint_color);
+				}
+
```

Comment:
Screenshot? :)

Reply:
there you go
![screenshot from 2017-09-15 16-05-03](https://user-images.githubusercontent.com/1103897/30503543-3f43a984-9a30-11e7-9a70-5688f2c07a02.png)